Bottom: 11733402ecb6e8f19b2ca2972cd200304643c8c6
Top:    5e84ba4ae234e08810f6e04608af4b87d3b2993b
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2015-01-11 17:56:03 +0000

Refresh of main-c-initialise-crosshair

---

diff --git a/src/main.c b/src/main.c
index 04a116c..58ba024 100644
--- a/src/main.c
+++ b/src/main.c
@@ -1990,8 +1990,7 @@ main (int argc, char *argv[])
 
   ResetStackAndVisibility ();
 
-  if (gui->gui)
-    InitCrosshair ();
+  InitCrosshair ();
   InitHandler ();
   InitBuffers ();
   SetMode (ARROW_MODE);
